### DISCUSSION TITLE
Help request: on the cover the 1st adviser is printed with slight spacing UMINHO

### DISCUSSION THREAD

**Discussion by Marcelo-Alves7** [joaomlourenco/novathesis#277] [Q&A] [answered]
Hello professor, I have been using the template to write my PhD thesis at UM (engineering). However I updated the document to the new version and noticed a small aspect (which did not happen in version [6.4.3]). When I put the advisers and co-advisors' names, the 1st printed name (adviser) is inserted in the cover with a slight horizontal spacing, which should not happen. In the case of co-advisers this problem no longer occurs. I send image of the question.

![image](https://user-images.githubusercontent.com/98588105/151536583-d70250e0-2996-4390-a5fc-85390fdca2be.png)

I tried to see if I could understand how to fix it, checking the file novathesis.cls and uminho-defaults.ldf, but without success. The part of the spacing and orientation on the page I could not identify.
Thank you in advance for your help and thank you for this important work.

**Comment by joaomlourenco** [comment, marked as answer]
Created and issue from this discussion.

**Comment by joaomlourenco** [comment]
Fixed in the last commit (09fa66a).

**Reply by Marcelo-Alves7** [reply]
Thank You professor! It Works :)
